src/cards/mana_steal.ts

```diff
@@ -21,18 +22,18 @@ const spell: Spell = {
     supportQuantity: true,
     manaCost: 0,
     healthCost: health_burn,
-    expenseScaling: 1,
+    expenseScaling: 2,
     probability: probabilityMap[CardRarity.SPECIAL],
     thumbnail: 'spellIconManaSteal.png',
-    description: ['spell_mana_steal', mana_stolen.toString()],
+    description: ['spell_mana_steal', base_mana_stolen.toString()],
     effect: async (state, card, quantity, underworld, prediction) => {
       // .filter: only target living units
       const targets = state.targetedUnits.filter(u => u.alive && u.mana > 0);
       const caster = state.casterUnit;
       let promises = [];
       let totalManaStolen = 0;
       for (let unit of targets) {
-        const unitManaStolen = Math.min(unit.mana, mana_stolen * quantity);
+        const unitManaStolen = Math.min(unit.mana, Math.floor(base_mana_stolen * quantity / (((targets.length - 1) / max_mana_stolen_mult) + 1)));
```

Comment:
I'm hesitant about this one for reasons I mentioned earlier.  I'll test it out in game and see how it feels.

Reply:
Similar to the balancing I recently did to Target Similar, it needs to allow "super awesome ridiculous plays" but it shouldn't be too easy to do so.  I'm all for sensible balancing of the spells that are too easy to use powerfully but we must maintain that Spellmasons is a game where you can take magic to absurd levels - that's core to it's identity.
Let's talk more about it next time we chat

Reply:
Understandable. I'm honestly not 100% on board with this change either. Balance wise, it feels awesome, but the formula is a little arbitrary and unintuitive, I think. It's not something you can really understand or calculate quickly without a graph in front of you.

Ideally, it'd be as easy to understand as "steal 20 mana from every target", while also not being so strong it makes all other game strategies redundant. My main problem with mana steal isn't the absurdity of it, but the fact that it's so easy to execute. Imo it feels more like a cop out or cheat than a ridiculously awesome and creative play in its current state, and that's primarily what I want to fix.

Reply:
Yeah, I feel that.
I tried this one out and it feels really nerfed now.  Lets revert this for now and put a pin in it to visit later.